### DISCUSSION TITLE
Plotting with dual wallets on the main node

### DISCUSSION THREAD

**Discussion by pluto2021** [Chia-Network/chia-blockchain#5664] [Support]
When you have two wallets at the main node, when logging in using either private key, existing plots are shown as valid plots. Is this expected? How to figure out which key each plot belongs to? or it does not matter as long as both wallets are on a same machine?

Also how to keep both wallet synced?

**Comment by lordzamura** [comment]
Gives you an example.
You have 1 wallet and 3 computers.
You can sign in to your wallet all of 3 computers.
You can plot. And you can merge you plots in one place.
Plots signed on wallet key.

**Comment by BrandtH22** [comment]
@pluto2021 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
